error_handler/build.gradle

```diff
@@ -0,0 +1,116 @@
+buildscript {
+    ext {
+        junitVersion = '5.3.2'
+        springFoxVersion = '2.9.2'
+        jacksonVersion = '2.9.8'
+    }
+}
+
+plugins {
+    id 'io.franzbecker.gradle-lombok' version '1.14'
+    id "org.owasp.dependencycheck" version "4.0.0.1"
+    id "com.github.spotbugs" version "1.6.5"
+    id 'pmd'
+    id 'checkstyle'
+}
+
+apply plugin: 'java'
+
+lombok {
+    version = '1.18.4'
+    sha256 = ""
+}
+
+group = 'uk.gov.dhsc.htbhf'
+sourceCompatibility = 1.11
+targetCompatibility = 1.11
+
+repositories {
+    mavenCentral()
+    maven {
+        url  "https://dl.bintray.com/departmentofhealth-htbhf/maven"
+    }
+}
+
+
+dependencies {
+    implementation "org.springframework:spring-web:${springVersion}"
+    implementation "org.springframework.boot:spring-boot-autoconfigure:${springBootVersion}"
```

Comment:
I'm not keen on having anything Spring Boot in this common module, is this essential? In my opinion, common libraries should never control and mandate Spring Boot dependancies, it only leads to real problems in projects that use it.

Reply:
This is needed for the ConditionalOnMissingBean, which could be taken out as there's only one bean in the configuration.

Reply:
Removed.